packages/azure-themes/src/azure/styles/DropDown.styles.ts

```diff
@@ -122,34 +136,53 @@ export const DropdownStyles = (props: IDropdownStyleProps): Partial<IDropdownSty
     dropdownItem: {
       color: semanticColors.bodyText,
       fontSize: theme.fonts.medium.fontSize,
+      backgroundColor: extendedSemanticColors.controlBackground,
       selectors: {
         // active: Mouse down on the item, then drag outside.
         '&:hover, &:active, &:hover:focus': {
           backgroundColor: semanticColors.listItemBackgroundHovered,
           borderColor: 'transparent',
-          color: semanticColors.buttonTextHovered,
+          color: extendedSemanticColors.commandBarButtonTextHover,
         },
       },
     },
     dropdownItemDisabled: {
-      color: semanticColors.disabledBodyText,
+      color: semanticColors.primaryButtonTextDisabled,
       fontSize: theme.fonts.medium.fontSize,
+      backgroundColor: extendedSemanticColors.controlBackground,
     },
     dropdownItemSelected: {
       fontSize: theme.fonts.medium.fontSize,
       backgroundColor: semanticColors.listItemBackgroundChecked,
-      color: semanticColors.bodyText,
+      color: extendedSemanticColors.commandBarButtonTextHover,
       selectors: {
         '&:hover, &:focus, &:active, &:hover:focus': {
           backgroundColor: semanticColors.listItemBackgroundChecked,
           borderColor: 'transparent',
-          color: semanticColors.bodyText,
         },
       },
     },
     dropdownItemHeader: {
+      fontSize: theme.fonts.large.fontSize,
```

Comment:
Just checked the Figma file and I see 13px for dropdown header text. Can you let me know what this change resulted in and if its indeed 13px? It might be a good idea to start documenting in your PR what the font size change is exactly! (ex: 13px > 16px). 


![image](https://user-images.githubusercontent.com/30805892/114617238-bb67d300-9c5c-11eb-8b32-59c401c74d00.png)

Reply:
> you let me know what this change resulted in and if its indeed 13px?

This was not needed. I took out the item header size.